src/util/Store.js

```diff
@@ -7,7 +7,7 @@ export class Store {
 
 	constructor(value) {
 		// wrap primitives with a generic "value" field to generate getter and setter
-		this._singlePropertyMode = !isObjectLike(value);
+		this._singlePropertyMode = !isUndefined(value) && !isObjectLike(value);
```

Comment:
@quarkus It is probably a real edge case scenario - but what if I wanted my single property store to be initialized with `undefined` ? How about we simply check if any argument was given with `arguments.length === 0`